docs/HowTo/PipelineAutomation.md

```diff
@@ -0,0 +1,55 @@
+---
+description: This page provides guidance to the teams in BC Gov that look to get started with encorporating Continuous integration (CI) and continuous delivery (CD) into the lifecycle of their application hosted on the Private Cloud Openshift Platform. 
+title: CI/CD with Pipeline Automation
+tags:
+  - continuous integration
+  - continuous deployment
+  - pipeline
+  - template
+  - ArgoCD
+  - Tekton
+  - Openshift Pipelines
+  - GitHub Actions
+---
+
+# Use of CI/CD Pipeline Templates for Private Cloud teams
+
+So you have now built a new and shiny app and want to deploy it to the BC Gov's Private Cloud Openshift Platform. You also want to take advantage of the Continuous integration (CI) and Continuous delivery (CD) power that can enable your team to automatically build, test, and prepare code changes for release to production so that your software delivery is more efficient, rapid and - **very importantly** - secure.
```

Comment:
```suggestion
So now you have built a new and shiny app and want to deploy it to the BC Gov's Private Cloud Openshift Platform. You also want to take advantage of the Continuous integration (CI) and Continuous delivery (CD) power that can enable your team to automatically build, test, and prepare code changes for release to production so that your software delivery is more efficient, rapid and - **very importantly** - secure.
```
Minor grammar fix.